webpack/assets/javascripts/react_app/components/common/Icon/Icon.consts.js

```diff
@@ -4,6 +4,7 @@ const infoIcon = 'pficon pficon-info';
 const warningIcon = 'pficon pficon-warning-triangle-o';
 const errorIcon = 'pficon pficon-error-circle-o';
 const closeIcon = 'pficon pficon-close';
+const questionSign = 'glyphicon glyphicon-question-sign';
```

Comment:
From the patternfly [icons page](https://www.patternfly.org/styles/icons/):

>  Bootstrap Glyphicons are also included in PatternFly, but are not recommended for use

Maybe use `fa fa-question`?

Reply:
that's fine - I was keeping the original behavior.